android/app/src/main/java/com/pillarproject/wallet/MainApplication.java

```diff
@@ -61,6 +62,7 @@
       return Arrays.<ReactPackage>asList(
         new ColorMatrixImageFiltersPackage(),
         new MainReactPackage(),
+            new RNGestureHandlerPackage(),
```

Comment:
Can we fix this indentation?

Reply:
![image](https://user-images.githubusercontent.com/5252494/66653860-9fd3d480-ec41-11e9-882f-f3fd8166c460.png)